src/core/stream/stream_loader.ts

```diff
@@ -194,6 +220,27 @@ export default function StreamLoader({
     const stallingManager$ = StallingManager(mediaElement, clock$)
       .pipe(map(EVENTS.stalled));
 
+    // Creates an observable which will refresh the manifest after
+    // the update period has elapsed.
+    const updateManifest$ = refreshMinimumUpdatePeriod$.pipe(
+      mergeMap((lastUpdatePeriod) => {
```

Comment:
Not a fan of this variable's name. It's actually the maximum refresh timeout of the manifest right here.

Reply:
Also, shouldn't it be a ``switchMap``?